identifiers/country-gr/electoral_districts.csv

```diff
@@ -0,0 +1,60 @@
+id,name
+ocd-division/country:gr/ed:a~_anatolikis_attikis,A' Anatolikis Attikis
```

Comment:
For all these, can we put "A" (B1, etc.) at the end, without the apostrophe?

Reply:
Of course we can. Done :)

Reply:
Thank you - can the ID also be correspondingly updated? e.g. `anatolikis_attikis_a` instead of `a~_anatolikis_attikis`.

Reply:
IDs updated correspondingly to updated names.